.github/workflows/python-main.yml

```
@@ -743,17 +714,29 @@ jobs:
           $env:EXPECTED_VERSION = $ver
           python bindings/python/ci/smoke_depthai.py
-      - name: Upload combined wheels to artifactory
-        if: github.event_name == 'workflow_dispatch' || github.event_name == 'push'
-        run: cd bindings/python && bash ./ci/upload-artifactory.sh --snapshot
+
+  upload-wheels-to-ak:
+    if: github.event_name == 'workflow_dispatch' || github.event_name == 'push'
+    needs: [combine-macos-wheels, combine-linux-x86_64-wheels, combine-linux-arm64-wheels, combine-windows-x86_64-wheels]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)